src/Mvc/Mvc.Core/src/CreatedResult.cs

```diff
@@ -64,19 +60,10 @@ public CreatedResult(Uri location, object? value)
     /// <summary>
     /// Gets or sets the location at which the content has been created.
     /// </summary>
-    public string Location
```

Comment:
When approving the API in https://github.com/dotnet/aspnetcore/issues/39454#issuecomment-1012457232, we did not consider that this property would need to be made nullable. I'll update the API proposal and see if everyone still thinks we should change MVC's `CreatedResult` or if it's too breaking.

We can work out the `PublicAPI.Unshipped.txt` changes to get the build to pass. It gets complicated when changing the nullability of shipped APIs. We might have to bypass the build check when changing the nullablity of a property setter like this.

Reply:
@BrennanConroy  When we update the shipped.txt file we got the following error in build.

![image](https://user-images.githubusercontent.com/83835739/178041716-0010f44f-9698-4883-b58d-9704467b8123.png)

Reply:
That's expected, someone will be able to override that and merge the PR